x-pack/plugins/ml/server/models/data_recognizer/data_recognizer.js

```diff
@@ -155,25 +171,33 @@ export class DataRecognizer {
     const kibana = {};
     // load all of the job configs
     await Promise.all(manifestJSON.jobs.map(async (job) => {
-      const jobConfig = await this.readFile(`${this.modulesDir}/${dirName}/${ML_DIR}/${job.file}`);
-      // use the file name for the id
-      jobs.push({
-        id: `${prefix}${job.id}`,
-        config: JSON.parse(jobConfig)
-      });
+      try {
+        const jobConfig = await this.readFile(`${this.modulesDir}/${dirName}/${ML_DIR}/${job.file}`);
+        // use the file name for the id
+        jobs.push({
+          id: `${prefix}${job.id}`,
+          config: JSON.parse(jobConfig)
+        });
+      } catch (error) {
```

Comment:
Would it be useful to log out the actual error being caught somewhere?

Reply:
Yes, good point. I'll append the error to these log messages.